models/migrations/20190829130000_user_categories.js

```diff
@@ -0,0 +1,18 @@
+exports.up = function(knex) {
+  return knex.schema.createTable('users_categories', tbl => {
+    tbl.increments();
+    tbl
+      .foreign('to_categories')
+      .references('categories_id')
+      .inTable('categories');
+    tbl
+      .foreign('to_users')
+      .references('users_id')
+      .inTable('users');
+    tbl.timestamp('created_at', { useTz: true });
+  });
+};
+
```

Comment:
Taken from the docs:
`table.foreign(column).references(column).inTable(table)`
`A default key name using the columns is used unless foreignKeyName is specified. `

I *THINK* you may have to use the same name in foreign as references and you can foreignKeyName to rename if you want.  Although if it works, it works

Reply:
so am I taking out .foreign()?